pkg/agent/proxy/proxier_test.go

```diff
@@ -2184,9 +2175,11 @@ func testServiceInternalTrafficPolicyUpdate(t *testing.T,
 	fp.serviceChanges.OnServiceUpdate(svc, updatedSvc)
 	groupIDLocal := fp.groupCounter.AllocateIfNotExist(svcPortName, true)
 
-	mockOFClient.EXPECT().InstallEndpointFlows(bindingProtocol, gomock.InAnyOrder(expectedLocalEps)).Times(1)
 	mockOFClient.EXPECT().UninstallEndpointFlows(bindingProtocol, expectedRemoteEps).Times(1)
+	mockOFClient.EXPECT().UninstallServiceGroup(groupID).Times(1)
+	mockOFClient.EXPECT().UninstallServiceFlows(svcIP, uint16(svcPort), bindingProtocol).Times(1)
 	mockOFClient.EXPECT().InstallServiceGroup(groupIDLocal, false, expectedLocalEps).Times(1)
+	mockOFClient.EXPECT().InstallServiceFlows(groupIDLocal, svcIP, uint16(svcPort), bindingProtocol, uint16(0), false, corev1.ServiceTypeClusterIP, false).Times(1)
```

Comment:
The uninstall and install validate the 3rd bug "After updating InternalTrafficPolicy, the flows for ClusterIP isn't updated" is fixed.